build/commands/lib/config.js

```diff
@@ -259,8 +269,11 @@ Config.prototype.buildArgs = function () {
   }
 
   if (process.platform === 'darwin') {
-    args.use_system_xcode = false
-    args.mac_sdk_official_version = "11.3"
+    const sdkVersion = getMacOSSDKVersion()
+    if (sdkVersion) {
+      args.use_system_xcode = false
```

Comment:
We should not be doing this for either of these, especially for `mac_sdk_official_version`. These changes should be reverted

Reply:
sorry, actually the problem is that the original block of code was wrong and needs to go away